### PR TITLE
Jvenstad/access control allow api prefix 

### PR DESCRIPTION
**Pull request by jonmv**
@tokle please review and merge.

### CODE REVIEW

File: controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/user/UserApiTest.java [hunk 1]
```diff
@@ -120,6 +125,11 @@ public void testUserManagement() {
                                       .roles(Set.of(roles.tenantOperator(id.tenant()))),
                               new File("application-roles.json"));
 
+        // GET application role information is available also under the /api prefix.
+        tester.assertResponse(request("api//user/v1/tenant/my-tenant/application/my-app")
```
Comment (tokle) [inline controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/user/UserApiTest.java]: did you intend to do api//user or should this be /api/user?

Comment (jonmv) [reply controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/user/UserApiTest.java]: What, how could this pass?

Comment (jonmv) [reply controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/user/UserApiTest.java]: Some normalisation is probably done, then. No, I didn't intend for this.

Comment (jonmv) [reply controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/user/UserApiTest.java]: So this was interpreted as just `/user` — the `8080api` port was interpreted as port 8080. Nice.

**Review by tokle — APPROVED** [review]

**Comment by jonmv** [comment]
Urgh, pushed to wrong branch :)

**Review by tokle — APPROVED** [review]

**Pull request merged**